endoreg_db/data/requirement/lab_value.yaml

```
@@ -364,3 +364,78 @@
+    name: "lab_value_inr_normal"
+    name_de: "INR im Normalbereich"
+    name_en: "INR normal"
+    description: "Checks if the latest INR is within the normal range."
+    requirement_types:
+      - "patient"
+      - "patient_lab_sample"
+      - "patient_lab_value"
+    operators:
+      - lab_latest_numeric_normal
+    lab_values:
+      - "international_normalized_ratio"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)